types/styletron-engine-monolithic/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for styletron-engine-monolithic 0.0
+// Project: https://github.com/styletron/styletron
+// Definitions by: Tom Golden <https://github.com/tbjgolden>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+import { KeyframesObject, FontFace as FontFaceObject, StandardEngine, StyleObject } from 'styletron-standard';
+
+export class SequentialIDGenerator {
```

Comment:
those are not exported symbols, how they could be here?
https://github.com/styletron/styletron/blob/master/packages/styletron-engine-monolithic/src/index.js#L3-L4

Reply:
ah - my bad; should have used declare here, 1 sec

Reply:
that''s OK, you can eventually export types (interface/type) if that is somehow useful when consuming this package.